test/security_test.rb

```diff
@@ -241,6 +241,8 @@ def enabled?
 
     describe "#write_kernel_settings" do
       before do
+        # there are some failure with running uname in chroot without root perms
+        allow(Yast::Report).to receive(:Error)
```

Comment:
I think we should rather skipp calling `uname` in the tests than mock the error popup. This might hide some other issues in the code or test. Moreover we should avoid executing some real commands in the system.

Reply:
it is not so easy as it is hidden very deep in sysctl class - https://github.com/yast/yast-yast2/blob/1629c3a520ebd8e8b93f1161f06d9e140d65b9ec/library/general/src/lib/cfa/sysctl_config.rb#L183
and it handles well if it failed